### ISSUE TITLE
Sprint Epic 12th - 26th January

### ISSUE THREAD

**Issue by ErisDS** [TryGhost/Ghost#6328]
## New editor demo: 

Over the next 2 weeks, we want to get the demo to a point where we can use it to determine our next steps. This will mostly be @kevinansfield:

- [ ] working in labs and allowing users to edit old & new content (old content via loading in a markdown card) - **Update:** partially completed
- [x] initial implementation of a markdown card
- [x] initial UI for adding and removing cards (meaning content sections when editing a post, not installing/uninstalling cards :wink:)
- [ ] using the implementation of the markdown card to come up with options for allowing custom cards to be installed/uninstalled - **Update:** adding CodeMirror card to further flesh out card addon requirements

## Post scheduling:

Over the next 2 weeks, I (@erisds) will be putting together specs for the UI and a pluggable backend to unblock post scheduling - one of the [most wanted](http://ideas.ghost.org/forums/285309-wishlist/filters/top) features. Development will likely not start until the next sprint.

- [x] Spec for the UI
- [x] Spec for the backend (pluggable)

## Hashtags:

The majority of the functionality for hashtags is ready and waiting in PR form, we just need to complete some data migration work (which is also needed for the new editor demo). We should be getting all this merged in by the end of this sprint.

- [ ] Data migration #6301 
- [ ] First working version in labs #6165

**Comment by ErisDS** [comment]
Closing in favour of #6470
